test/outside/off/off_shops_manager_test.dart

```diff
@@ -294,13 +305,33 @@ void main() {
   test('fetch barcodes when could not fetch shops', () async {
     await _initShops(
       cameraPos: Coord(lat: 10, lon: 10),
-      addressOfAnyCoords: Ok(OsmAddress((e) => e.countryCode = 'be')),
+      addressOfAnyCoords: Ok(OsmAddress((e) => e.countryCode = BELGIUM)),
       offApiShops: Err(OffRestApiError.OTHER), // ERROR!!!!!!!!!!!!
     );
 
     final shop = someOffShops.first;
     _initOffShopBarcodes(
-      countryCode: 'be',
+      countryCode: BELGIUM,
+      veganIngredientsBarcodes: {
+        shop: ['123', '124']
+      },
+    );
+
+    final fetchedBarcodesRes = await offShopsManager
+        .fetchVeganBarcodesForShops({shop.name!}, [LangCode.be]);
+    expect(fetchedBarcodesRes.isErr, isTrue);
+  });
+
+  test('fetch barcodes for country not in allowedList', () async {
```

Comment:
Similarly, I think this test probably should also check whether an empty collection is returned rather than an error...

Reply:
Yeah i'll fix the tests when I reworked the code. Thx for the review!